tests/parametric/test_headers_tracecontext.py

```diff
@@ -713,6 +713,100 @@ def test_tracestate_duplicated_keys(self, test_agent, test_library):
         assert traceparent4.trace_id == "12345678901234567890123456789012"
         assert "foo=1" in str(tracestate4) or "foo=2" in str(tracestate4)
 
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "php", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "cpp", reason="Not implemented")
+    @missing_feature(context.library == "ruby", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    def test_tracestate_w3c_p_extract(self, test_agent, test_library):
```

Comment:
nit: Are we not parametrizing this test because of case 3?

Reply:
Currently this feature is not merged and released in any tracers so we have to skip this test. The plan is to enable these tests by the EOQ

Reply:
Case 3 doesn't have anything to do with this